dangerzone/startup.py

```diff
@@ -168,6 +169,47 @@ def run(self) -> None:
             raise RuntimeError("Failed to stop all other running Podman machines.")
 
 
+class WSLInstallTask(Task):
+    name = "Installing Windows Subsystem for Linux"
+
+    def should_skip(self) -> bool:
+        return platform.system() != "Windows" or wsl.is_wsl_installed()
```

Comment:
since we use a namespace, let's simplify the name of the functions:

```suggestion
        return platform.system() != "Windows" or wsl.is_installed()
```